### PR TITLE
Add 60_ansi_arrow_split_bs_7u_spc layout & keymap

### PR DESCRIPTION
**Pull request by mrsendyyk**
## Description

60 ANSI Arrow Split Backspace & 7U Space Layout and Keymap

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* None

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

File: layouts/default/60_ansi_arrow_split_bs_7u_spc/layout.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[{a:7},"","","","","","","","","","","","","","",""],
+[{w:1.5},"","","","","","","","","","","","","",{w:1.5},""],
+[{w:1.75},"","","","","","","","","","","","",{w:2.25},""],
+[{w:2.25},"","","","","","","","","","",{w:1.75},"","",""],
+[{w:1.5},"","",{w:1.5},"",{w:7},"","","",""]
```
Comment (noroadsleft) [inline layouts/default/60_ansi_arrow_split_bs_7u_spc/layout.json:5]:
```suggestion
[{w:1.5},"","",{w:1.5},"",{w:7},"","","","",""]
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]

File: layouts/default/60_ansi_arrow_split_bs_7u_spc/layout.json (hunk 1 above)
Comment (mrsendyyk) [reply layouts/default/60_ansi_arrow_split_bs_7u_spc/layout.json:5]: Done

**Review by mrsendyyk — COMMENTED** [review]
Ready for review @mechmerlin

**Review by noroadsleft — APPROVED** [review]

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**